pyilluminate/illuminate.py

```diff
@@ -53,6 +58,51 @@
 # https://forum.pjrc.com/threads/54773-Inreasing-USB-Serial-Buffer-Teensy-3-2
 MAX_ARGUMENT_CHAR_COUNT = 64 * 1
 
+thread_lock = RLock()
```

Comment:
This won't work with multiple different LED boards at the same time.

I don't think we need to lock 'static" functions. They don't use the serial port itself. They just ask the operating system for the status of the USB port.